GoogleDataTransport/GDTCORLibrary/Public/GDTCOREvent.h

```diff
@@ -73,17 +74,13 @@ typedef NS_ENUM(NSInteger, GDTCOREventQoS) {
 /** Bytes that can be used by an uploader later on. */
 @property(nullable, nonatomic) NSData *customBytes;
 
-// Please use the designated initializer.
-- (instancetype)init NS_UNAVAILABLE;
-
 /** Initializes an instance using the given mappingID.
  *
  * @param mappingID The mapping identifier.
  * @param target The event's target identifier.
  * @return An instance of this class.
  */
-- (nullable instancetype)initWithMappingID:(NSString *)mappingID
-                                    target:(NSInteger)target NS_DESIGNATED_INITIALIZER;
+- (nullable instancetype)initWithMappingID:(NSString *)mappingID target:(GDTCORTarget)target;
```

Comment:
Should `mappingID` be nullable as well?

Reply:
It shouldn't be nullable in the initializer, but if you call `-init` it'll produce a nil value when you call the getter.